mobile/Screen/SubmissionScreen.js

```diff
@@ -1,20 +1,198 @@
+import * as ImagePicker from 'expo-image-picker';
+
+const { URL } = process.env;
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: '#fff',
     alignItems: 'center',
     justifyContent: 'center',
+    paddingTop: 10,
+  },
+  scrollView: {
+    marginHorizontal: 20,
+  },
+  input: {
+    height: 40,
+    margin: 12,
+    borderWidth: 1,
+    padding: 10,
+  },
+  submissionsContainer: {
+    padding: 5,
+  },
+  submission: {
+    margin: 10,
   },
 });
 
 function SubmissionScreen() {
+  const [submissions, setSubmissions] = useState([]);
+
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [platform, setPlatform] = useState('');
+  const [accountTag, setAccountTag] = useState('');
+  const [artworkTitle, setArtworkTitle] = useState('');
+  const [description, setDescription] = useState('');
+
+  const [images, setImages] = useState([]);
+
+  function clearInput() {
+    setName('');
+    setEmail('');
+    setPlatform('');
+    setAccountTag('');
+    setArtworkTitle('');
+    setDescription('');
+  }
+
+  const submit = async () => {
+    try {
+      const result = await axios.post(`${URL}/submissions/post`, {
+        name,
+        email,
+        socials: {
+          platform,
+          tag: accountTag,
+        },
+        title: artworkTitle,
+        description,
+      });
+      setSubmissions((prev) => ([...prev, result.data]));
+      clearInput();
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  const pickImage = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    if (!result.canceled) {
+      setImages((prev) => ([...prev, result.assets[0].uri]));
+    }
+  };
+
+  const openCamera = async () => {
+    const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+
+    if (permissionResult.granted === false) {
+      return;
+    }
+
+    const result = await ImagePicker.launchCameraAsync();
+
+    if (!result.canceled) {
+      setImages((prev) => ([...prev, result.assets[0].uri]));
+    }
+  };
+
+  useEffect(() => {
+    (async () => {
+      const result = await axios.get(`${URL}/submissions/get`);
+      setSubmissions(result.data);
+    })();
+  }, []);
+
   return (
-    <View style={styles.container}>
-      <Text>Submission Screen</Text>
-    </View>
+    <ScrollView>
+      <View style={styles.container}>
+        <View>
+          <Button title="Image" onPress={pickImage} />
+          <Button title="Camera" onPress={openCamera} />
+          {images.map((src) => <Image source={{ uri: src }} style={{ width: 200, height: 200 }} />)}
+          <TextInput
+            styles={styles.input}
+            placeholder="Name"
+            onChangeText={(newName) => setName(newName)}
+            defaultValue={name}
+          />
```

Comment:
You will likely need to add a reference later to allow users to hit enter on the keyboard and switch to the next input box